### DISCUSSION TITLE
Refresh package on NPM assets

### DISCUSSION THREAD

**Discussion by csakiistvan** [tomik23/circular-progress-bar#96] [Announcements]
Hi Grzegorz,

I would like to use your package but the latest version(s) are not in the list on https://asset-packagist.org/package/npm-asset/tomickigrzegorz--react-circular-progress-bar, can you help me please on this?

Thanks,
István

**Comment by tomickigrzegorz** [comment]
Hi,
This is not the React version. You are on the vanilla version. The vanilla version does not have npm.
Here is the react version https://github.com/tomickigrzegorz/react-circular-progress-bar but it is not updated and it has npm.
I don't have time to change the libraries.
